xtc/src/ledger.rs

```diff
@@ -153,32 +152,31 @@ pub struct ErrorDetails {
     code: APIError,
 }
 
-static InsufficientBalanceError: ErrorDetails = ErrorDetails {
+const INSUFFICIENT_BALANCE_ERROR: ErrorDetails = ErrorDetails {
```

Comment:
we also have some type definitions in the candid files(apart from functions), so if you change a type, let's say add a new field to a struct or change struct name, we need to make sure we talk to upstream/make idl change. 

Just leave it here as a note, fortunately those error types are not exposed yet to the upstream.

Reply:
Interesting, didn't know that. Good thing they were renamed to follow naming guidelines before that became an issue.